app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,60 @@
+# encoding: utf-8
+
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  #if Rails.env.production? || Rails.env.development?
```

Comment:
Incorrect indentation detected (column 2 instead of 4).<br>Missing space after #.